test/audits/manifest/icons-192.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const Audit = require('../../../audits/manifest/icons-192.js');
+const assert = require('assert');
+const manifestSrc = JSON.stringify(require('./manifest.json'));
+const manifestParser = require('../../../helpers/manifest-parser');
+const manifest = manifestParser(manifestSrc).value;
+
+/* global describe, it*/
+
+describe('manifest: icons-192 audit', () => {
+  it('fails when no manifest present', () => {
+    return assert.equal(Audit.audit({}).value, false);
+  });
+
+  it('fails when an empty manifest is present', () => {
+    return assert.equal(Audit.audit({manifest: {}}).value, false);
+  });
+
+  it('fails when a manifest contains no icons', () => {
+    const inputs = {
+      manifest: {
+        icons: null
+      }
+    };
+
+    return assert.equal(Audit.audit(inputs).value, false);
+  });
+
```

Comment:
if we want to be super thorough, we should probably add a test for an icon without a size and an icon with multiple sizes and 192x192 is in there somewhere. I can add those really fast